packages/core/src/storage/compare.ts

```diff
@@ -78,6 +79,16 @@ export class StorageLayoutComparator {
     }
   }
 
+  getVisibilityChange(original: ParsedTypeDetailed, updated: ParsedTypeDetailed): TypeChange | undefined {
+    const re = /(internal|external|public|private)/
```

Comment:
Better to match more strictly what we assume we will see:

```suggestion
    const re = /^t_function_(internal|external)/
```